person.rb

```diff
@@ -0,0 +1,32 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  @id_tem = 0
+
+  class << self
+    attr_accessor :id_tem
+  end
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    self.class.id_tem += 1
+    @id = self.class.id_tem
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    if of_age?
+      true
+    else
+      @parent_permission ? true : false
+    end
+  end
+
+  private
+
+  def of_age?
+    @age > 18
+  end
```

Comment:
- [ ] @Anand8317 Great work on your person class, however the ` of_Age?` method should return true if the user is 18 years and above in your case its checking if the user is above 18 years.
- [ ] [optional] You can also reduce the size of your `can_use_services` method by simply changing it to `def can_use_services? 
    of_age?() || @parent_permission
  end
`

Reply:
hi @danmainah, '@age > 18' returns true or false after checking the condition. So of_age? returns either true or false.
Thank you for the 2nd point.